docs/contributing/editing-native-code.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 99
+---
+
+# Editing Native Code
+
+If you just open one of the native files in the modules directly, you'll probably see a lot of red squiggles. This is
+because the native code is written in Kotlin and Swift, and they depend on other native modules.
+
+To edit these modules with the full git context of the project, you'll need to open the example app in Xcode and/or
+Android gitStudio, then open the native files from there.
```

Comment:
```suggestion
Android Studio, then open the native files from there.
```

Reply:
looks like I had the wrong window selected 😅